strictdoc/export/html/templates/components/form/field/contenteditable/field_with_human_title.jinja

```diff
@@ -0,0 +1,71 @@
+{%- assert field_class_name is defined, "field_class_name is defined" -%}
```

Comment:
We will do this a little differently when @mettta adds a realistic UI. Good for now.

Probably the `field_with_human_title` will be removed. Instead, the `row_with_reserved_field` will contain two fields, one for name, one for human title. @mettta will decide on the way.

Reply:
Sounds good! This was meant to provide the simplest UI possible with the corresponding functionality.